Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/SearchPageSettings.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from "react";
+import SettingsMenuContainer from "../../components/SettingsMenuContainer";
+import {
+  templateDefaults,
+  templateError,
+  TemplateUpdateProps
+} from "../../mainui/Template";
+import { routes } from "../../mainui/routes";
+import { Button, Grid } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import { languageStrings } from "../../util/langstrings";
+import {
+  CloudSettings,
+  defaultSearchSettings,
+  getCloudSettingsFromServer,
+  getSearchSettingsFromServer,
+  saveCloudSettingsToServer,
+  saveSearchSettingsToServer,
+  SearchSettings
+} from "./SearchSettingsModule";
+import { AxiosError, AxiosResponse } from "axios";
+import { generateFromError, generateNewErrorID } from "../../api/errors";
+import MessageInfo from "../../components/MessageInfo";
+import DefaultSortOrderSetting from "./components/DefaultSortOrderSetting";
+import GalleryViewsSettings from "./components/GalleryViewsSettings";
+import SettingsCheckbox from "../../components/SettingsCheckbox";
+
+let useStyles = makeStyles({
```

Comment:
useStyles doesn't appear to be reassigned, could it be made const?

```suggestion
const useStyles = makeStyles({
```